plugins/ksp/testData/api/typeAlias.kt

```diff
@@ -2,9 +2,12 @@
 // TEST PROCESSOR: TypeAliasProcessor
 // EXPECTED:
 // A = String
+// A = String
```

Comment:
This is a result of fixing `KSDeclarationContainer.declarations`, where `KSTypeAlias` WAS not included.